src/main/java/io/strimzi/kafka/access/internal/ParserException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access.internal;
+
+public class ParserException extends RuntimeException {
```

Comment:
could it be a more specific KafkaParserException?

Reply:
Since it could be an exception from either the Kafka or KafkaUser how about `KafkaResourceParceException`?

Reply:
The two resources specifically are `Kafka` and `KafkaUser`, those are both Strimzi resources but I was going off the name of the resources rather than the project. I could get on board with `StrimziResourceParseException`

Reply:
Discussed on the Community Call => `CustomResourceParseException` sounds like a good option.

Reply:
I've checked the java-operator-sdk and there are no existing exception classes that use this name or anything similar so I have made the update.